docs/io/optional/logging_configuration.ipynb

```diff
@@ -0,0 +1,512 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ad59aa05",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "download_atom_data('kurucz_cd23_chianti_H_He')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "64cf4bb2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sim = run_tardis(\"tardis_config_logger.yml\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9bf77405",
+   "metadata": {},
+   "source": [
+    "It can be examined that the logs are not printed.<br> The logging level, by default, is set to `CRITICAL`. Logs will only be captured if any `CRITICAL` level logs are encountered while running the simulation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "420dd7c2",
+   "metadata": {},
+   "source": [
+    "## Logging Configuration (Functional Arguments)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ac8d5088",
+   "metadata": {},
+   "source": [
+    "The `run_tardis()` function from the `tardis module` has two functional arguments : `log_state` & `specific`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d58f2d9c",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\" role=\"alert\">\n",
+    "    Both <strong>log_state</strong> & <strong>specific</strong> are optional arguments for the <strong>run_tardis()</strong> function, however, if <strong>specific</strong> argument is used then, <strong>log_state</strong> must be set to a particular level.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1176f370",
+   "metadata": {},
+   "source": [
+    "### `log_state`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c3547b04",
+   "metadata": {},
+   "source": [
+    "The `log_state` argument can be passed in `run_tardis()` to set the logging level for the simulation. The input for this argument **must be** one of the following: **Notset, Debug, Info, Warning, Error** or **Critical**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc2dcd71",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "sim = run_tardis(\"tardis_config_logger.yml\", log_state=\"Info\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52e778bc",
+   "metadata": {},
+   "source": [
+    "By setting up the `log_state` parameter to **\"Info\"** in the above example, we can check that the logs are at the **\"Info\"** or higher logging level. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25277fe8",
+   "metadata": {},
+   "source": [
+    "### `specific`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "56a5ef36",
+   "metadata": {},
+   "source": [
+    "The `specific` argument tells the logger to capture log messages set by the `log_state` parameter. It can only take *Boolean* values for input i.e, `True` or `False`. Take for example the following :"
```

Comment:
Can you remove the space before the colon.

Reply:
You need to fix the i.e., it should read "values for input, i.e., `True` or `False`. You need a comma before and after i.e.